spra-play-server/src/main/scala/net/wiringbits/spra/admin/repositories/daos/DatabaseTablesDAO.scala

```diff
@@ -230,7 +255,7 @@ object DatabaseTablesDAO {
   }
   def create(
       tableName: String,
-      fieldsAndValues: Map[TableColumn, String],
+      fieldsAndValues: Map[TableColumn, Serializable],
```

Comment:
Why `Serializable` trait?

Reply:
Because in the case of having an image column, we store images in the database as byte arrays. If we leave it as a String, incorrect information is stored.

Reply:
We should create a custom trait that extends Scala `Serializable` to be more descriptive and be more specific when typing